### ISSUE TITLE
Details of obtaining test results for each dataset

### ISSUE THREAD

**Issue by xiaodineu** [automl/auto-sklearn#563]
hi!
I have take a look at the results of the experiment on 140 datasets in the paper. I had two questions. 
1. Many datasets have columns of type string (object). How to deal with such columns? Direct abandonment or label-encoding?
2. How to get the test result in this paper,  train with all the columns (except target column) in the dataset as feature columns or manually specified some columns in datasets as feature columns.
3. I want to add some other metrics and datasets  in meta-learning. How should I start to get the new meta-data?

**Comment by mfeurer** [comment]
> Many datasets have columns of type string (object). How to deal with such columns? Direct abandonment or label-encoding?

To the best of my knowledge they only have columns of type category, in which the different categories can be replaced by integers. Please provide a dataset Id if there are datasets with string columns.

> How to get the test result in this paper, train with all the columns (except target column) in the dataset as feature columns or manually specified some columns in datasets as feature columns.

Sorry, but I don't get this comment. Do you want to know how we transformed the data from OpenML into X and y?

> I want to add some other metrics and datasets in meta-learning. How should I start to get the new meta-data?

Please have a look at [these scripts](https://github.com/automl/auto-sklearn/tree/master/scripts).

**Comment by xiaodineu** [comment]
1、I mean columns of type category.
2、Yes, because the MagicTelescope dataset [https://www.openml.org/d/1120](url). ID column have Strong correlation this class:, if X contains ID column we got accuracy of 100%
3、Got it, thanks!
